test/integration/downtime_with_invalid_dates_test.rb

```diff
@@ -0,0 +1,54 @@
+require "integration_test_helper"
+
+class DowntimeWithInvalidDates < ActionDispatch::IntegrationTest
+  setup do
+    setup_users
+
+    @edition = FactoryBot.create(
+      :transaction_edition,
+      :published,
+      title: "Apply to become a driving instructor",
+      slug: "apply-to-become-a-driving-instructor",
+    )
+
+    WebMock.reset!
+    stub_any_publishing_api_put_content
+    stub_any_publishing_api_publish
+  end
+
+  test "Scheduling new downtime with invalid dates" do
+    DowntimeScheduler.stubs(:schedule_publish_and_expiry)
+
+    visit root_path
+    click_link "Downtime"
+    click_link "Apply to become a driving instructor"
+
+    enter_start_time 1.day.ago
+    enter_end_time 1.day.ago - 1.day
+
+    click_button "Schedule downtime message"
+
+    assert page.has_link?("End time must be in the future", href: "#downtime_end_time")
+    assert page.has_link?("Start time must be earlier than end time", href: "#downtime_start_time")
+  end
+
+  def enter_start_time(start_time)
+    complete_date_inputs("downtime_start_time", start_time)
+  end
+
+  def enter_end_time(end_time)
+    complete_date_inputs("downtime_end_time", end_time)
+  end
+
+  def complete_date_inputs(input_id, time)
+    select time.year.to_s, from: "#{input_id}_1i"
+    select time.strftime("%B"), from: "#{input_id}_2i"
+    select time.day.to_s, from: "#{input_id}_3i"
+    select pad_digit_to_two_digits(time.hour.to_s), from: "#{input_id}_4i"
```

Comment:
This might be a more succinct way to achieve the same result, without needing to declare a new method for it.

```suggestion
    select "%02d" % time.hour, from: "#{input_id}_4i"
```

Whilst this might look quite tropical, it's actually just a [method called `%` defined on the `String` class](https://ruby-doc.org/core-2.2.0/String.html#method-i-25).

We pass it the `time.hour` integer as an input, and it returns a two-digit zero-padded string. Which also means we no longer need to manually call `.to_s` on the integer.